Sources/Datadog/Tracing/Span/SpanEncoder.swift

```diff
@@ -198,5 +198,8 @@ internal struct SpanEncoder {
             let metaKey = "meta.\($0.key)"
             try container.encode($0.value, forKey: DynamicCodingKey(metaKey))
         }
+        try span.userInfo.extraInfo.forEach {
+            try container.encode(EncodableValue($1), forKey: DynamicCodingKey("meta.usr.\($0)"))
```

Comment:
This leads to all spans being rejected by Intake if the user extra attribute is not `String`, e.g.:
```swift
Datadog.setUserInfo(
    id: "abcd-1234",
    name: "foo",
    email: "foo@example.com",
    extraInfo: [
        "is-registered": true
    ]
)
```
As stated in this method comment:
```swift
// NOTE: RUMM-299 only string values are supported for `meta.*` attributes
```
Whatever we encode on `meta.*` key must be serialised to `String` representation. We should use `JSONStringEncodableValue` in `SpanBuilder` as it is done for `span.tags`.